File: eFrizer/eFrizer.Model/Review/ReviewInsertRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eFrizer.Model
{
    public class ReviewInsertRequest
    {
        public int HairSalonId { get; set; }
        public int ClientId { get; set; }
        public int StarRating { get; set; }
    }
}

File: eFrizer/flutter_efrizer/lib/models/review/review_insert_request.dart
import 'package:flutter/material.dart';
import 'package:flutter_login/models/review/review.dart';

class ReviewInsertRequest{
  final int hairSalonId;
  final int clientId;
  final int starrating;

  ReviewInsertRequest({
    required this.hairSalonId,
    required this.clientId,
    required this.starrating,
  });

  Map<String, dynamic> toJson() => {
        "hairSalonId": hairSalonId,
        "clientId": clientId,
        "starRating": starrating,
      };
}